### PR TITLE
Update muon user and dev docs

### PR DESCRIPTION
**Pull request by AnthonyLim23**
**Description of work.**
This PR updates the user documents for the muon plotting and the dev docs for testing the muon GUI.

To test check:
- they build correctly
- the formatting looks ok
- spelling and grimmer

There is no associated issue.

This does not require release notes

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_5_MA2.rst [hunk 1]
```diff
@@ -23,17 +23,54 @@ Setup:
...
+----------------------------
+
+- Go to the **Corrections** tab
+- For the ``Background`` select ``Auto``
+- The plots will now average around 0
+- The values should be (approximatley):
+	- Forw: ``19.2``
+	- Back: ``93.6``
+	- Left: ``104.4``
+	- Rite: ``77.2``
```
Comment (SilkeSchomann) [inline dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_5_MA2.rst:41]:
```suggestion
	- Right: ``77.2``
```

File: dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_5_MA2.rst [hunk 2]
```diff
@@ -23,17 +23,54 @@ Setup:
...
+- Tick the ``use Raw`` box and the values will change
+- The new values should be similar (within 1) to before
+
+
+
+Flat Background Corrections:
+----------------------------
+
+- Change the ``Select Function`` to ``Flat Background``
+- The curves wil be curved upwards again and the table will report error
```
Comment (SilkeSchomann) [inline dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_5_MA2.rst:51]:
```suggestion
- The curves will be curved upwards again and the table will report an error
```

File: dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_5_MA2.rst [hunk 3]
```diff
@@ -23,17 +23,54 @@ Setup:
...
+- The **Corrections** tab will show a red astrix
+- Go to the **Home** tab
+- Unitck ``Time Zero``
```
Comment (SilkeSchomann) [inline dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_5_MA2.rst:54]:
```suggestion
- Untick ``Time Zero``
```

File: dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_5_MA2.rst [hunk 4]
```diff
@@ -23,17 +23,54 @@ Setup:
...
+- The new values should be similar (within 1) to before
+
+
+
+Flat Background Corrections:
+----------------------------
+
+- Change the ``Select Function`` to ``Flat Background``
+- The curves wil be curved upwards again and the table will report error
+- The **Corrections** tab will show a red astrix
```
Comment (SilkeSchomann) [inline dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_5_MA2.rst:52]:
```suggestion
- The **Corrections** tab will show a red asterisk
```

File: docs/source/interfaces/muon/muon_analysis_plotting_window.rst [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+- Results
+
+It displays the selected data for a fit and if a fit is requested this is also shown on the plot.
+In the sequential fitting tab, the user can select the data to plot by highlighting the different rows in the table.
+
+
+Model Data
+^^^^^^^^^^
+
+This shows the data that has been selected on the the **Model Fitting** tab.
```
Comment (SilkeSchomann) [inline docs/source/interfaces/muon/muon_analysis_plotting_window.rst:25]:
```suggestion
This shows the data that has been selected on the **Model Fitting** tab.
```

File: docs/source/interfaces/muon/muon_plotting_window.rst [hunk 6]
```diff
@@ -8,27 +8,36 @@ Plotting Window
...
 
-The workspaces plotted in the figure depends on the tab which is currently being viewed:
+This data can be displayed as a single plot, or as tiled plots.
 
-- If the user is on the Home, Grouping or Phase table tabs the workspaces corresponding to the loaded runs and
-  selected groups are plotted.
-- If the user is on the fitting tab, the workspace displayed in the workspace selection box is plotted.
-- If the user is on the sequential the plotted workspace is controlled through the table present in this tab.
+The custom plotting for Muon Analysis are described in the :ref:`here <Muon_Analysis_plotting-ref>`.
+The custom plotting for Frequency Doamin Analysis are described in the :ref:`here <Frequency_Domain_Analysis_plotting-ref>`.
```
Comment (SilkeSchomann) [inline docs/source/interfaces/muon/muon_plotting_window.rst:18]:
```suggestion
The custom plotting for Frequency Domain Analysis are described in the :ref:`here <Frequency_Domain_Analysis_plotting-ref>`.
```

File: docs/source/interfaces/muon/muon_plotting_window.rst [hunk 7]
```diff
@@ -8,27 +8,36 @@ Plotting Window
...
 **Plot type** Changes the plot type (e.g from asymmetry to counts).
 
 **Tiled plotting** The 'tile plots by' checkbox enables tiled plotting, which can be performed over runs or Groups/pairs.
+This is not always available.
 
 **Plot raw** Plot raw controls whether the raw or rebinned workspaces are plotted.
+This is not always available.
+
+**Plot difference** is only available for Fit Data and Model Data.
+When it is tiked it shows the difference between the data and the fit to the data.
```
Comment (SilkeSchomann) [inline docs/source/interfaces/muon/muon_plotting_window.rst:40]:
```suggestion
When it is ticked it shows the difference between the data and the fit to the data.
```

**Review by SilkeSchomann — CHANGES_REQUESTED** [review]
There are a few typos.

File: dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_5_MA2.rst (hunk 1 above)
Comment (AnthonyLim23) [reply dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_5_MA2.rst:41]: That is what the group is called, they are only able to write 4 letters.

**Review by SilkeSchomann — APPROVED** [review]
Documents build correctly and the formatting looks ok.

**Pull request merged**